### PR TITLE
[15.0][MIG] product_packaging_type_required: Migration to 15.0

### PR DESCRIPTION
**Pull request by AnnaPForgeFlow**
@ForgeFlow
Standard migration to 15.0

### CODE REVIEW

File: product_packaging_type_required/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Product Packaging Type Required",
+    "version": "15.0.1.1.0",
+    "development_status": "Beta",
+    "category": "Product",
+    "summary": "Product Packaging Type Required",
+    "author": "Camptocamp, " "Odoo Community Association (OCA)",
+    "maintainers": ["simahawk", "dcrier"],
+    "website": "https://github.com/OCA/product-attribute",
+    "license": "AGPL-3",
+    "depends": ["product", "product_packaging_type"],
```
Comment (DavidJForgeFlow) [inline product_packaging_type_required/__manifest__.py]: "product" can be erased as a dependency because "product_packaging_type" also depends on this

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM!

File: product_packaging_type_required/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Product Packaging Type Required",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.1.0",
```
Comment (rousseldenis) [inline product_packaging_type_required/__manifest__.py]: @AnnaPForgeFlow Please reset to 15.0.1.0.0

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: product_packaging_type_required/__manifest__.py (hunk 2 above)
Comment (AnnaPForgeFlow) [reply product_packaging_type_required/__manifest__.py]: @rousseldenis done!

Comment (rousseldenis) [reply product_packaging_type_required/__manifest__.py]: @AnnaPForgeFlow I don't think so :sweat_smile:

File: product_packaging_type_required/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Camptocamp (<http://www.camptocamp.com>).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Packaging Type Required",
+    "version": "15.0.1.1.0",
```
Comment (DavidJForgeFlow) [inline product_packaging_type_required/__manifest__.py]: @rousseldenis I see 15.0

Comment (rousseldenis) [reply product_packaging_type_required/__manifest__.py]: should be 15.0.1.**0**.0

Comment (AnnaPForgeFlow) [reply product_packaging_type_required/__manifest__.py]: @rousseldenis sorry, solved!

**Pull request closed without merging**